book/modules/appendix3.tex

```diff
@@ -220,12 +232,18 @@
 
 XXX Figure
 
-We can compute the determinant of the matrix $Q=[\xhat|\,\vec u_\theta]$ directly using the $2\times 2$ determinant formula.
```

Comment:
I changed the order and some of the wording of these passages because I thought this would make more sense.